.travis.yml

```diff
@@ -30,3 +45,32 @@ matrix:
   - jdk: openjdk11
   - jdk: openjdk12
   - jdk: openjdk13
+  
+  # deploys
+  - stage: deploy_edge
+    name: "Deploy Edge Release"
+    jdk: openjdk8
+  before_script:
+    - export PROJECT_MAVEN_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+    - export TRAVIS_TAG=${TRAVIS_TAG:-$PROJECT_MAVEN_VERSION+$(date +'%Y%m%d%H%M%S')-$TRAVIS_COMMIT}
+    script:
+      - mvn package -DskipTests -Dcheckstyle.skip=true
+      - docker build -f install/docker/Dockerfile -t airsonic/airsonic:latest -t airsonic/airsonic:edge/$TRAVIS_TAG .
+    before_deploy:
+      # Set up git user name and tag this commit
+      - git config --local user.name "Airsonic Travis"
+      - git config --local user.email "airsonic@tutanota.com"
+      - git tag $TRAVIS_TAG -m "Edge Release $TRAVIS_TAG"
+    deploy:
```

Comment:
Would this not pollute our releases page? I don't like the idea of users having to sift through hundreds of "nightly" builds to find actual releases.